Apps/package.json

```diff
@@ -3,10 +3,10 @@
   "version": "0.0.1",
   "private": true,
   "dependencies": {
-    "babylonjs": "5.0.0-alpha.60",
-    "babylonjs-gui": "5.0.0-alpha.60",
-    "babylonjs-loaders": "5.0.0-alpha.60",
-    "babylonjs-materials": "5.0.0-alpha.60",
+    "babylonjs": "5.0.0-alpha.65",
```

Comment:
When I tried upgrading to alpha.64 in another project earlier today, I saw XR was not working on iOS (solid background rendered instead of the camera frame). I guess you aren't seeing this behavior?

Reply:
I'm not seeing the behavior on OpenXR devices because our OpenXR and mobile XR implementations currently expect opposite clearing behavior on the JS side. This PR brings us back to the original behavior that changed due to a change on the JS side that looks for a specific type we don't currently define:
https://github.com/BabylonJS/BabylonNative/pull/964